plugins/a11y/sudo/src/index.ts

```diff
@@ -16,8 +16,6 @@ export function apply(ctx: Context) {
     .option('member', '-m [id:user]')
     .option('channel', '-c [id:channel]')
     .action(async ({ session, options }, message) => {
-      if (!message) return session.text('.expect-command')
-
```

Comment:
Why to delete these lines? I think it would be better to show this message instead of the default one that didn't explicitly notify which argument it lacks.
Instead, what about to register this command with `checkArgCount: false`?
Also, others commands everywhere that has the same problem as well.